afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayIntroText.kt

```diff
@@ -0,0 +1,17 @@
+package com.afterpay.android.view
+
+import com.afterpay.android.R
+
+enum class AfterpayIntroText(val resourceID: Int) {
+    NONE(R.string.afterpay_price_breakdown_string_empty),
+    PAY(R.string.afterpay_price_breakdown_intro_pay),
+    IN(R.string.afterpay_price_breakdown_intro_in),
+    OR(R.string.afterpay_price_breakdown_intro_or),
+    PAY_IN(R.string.afterpay_price_breakdown_intro_pay_in);
```

Comment:
`Make` is a newly added intro text in version `1.8.0` of the JS library. Should that be included here too?

Cc: @benfterpay

Reply:
Addressed in eaad376